.drone.yml

```diff
@@ -145,6 +145,20 @@ steps:
     depends_on:
       - build.Lint
 
+  - name: test.UnitTestRace
```

Comment:
not sure about the placement here. 

I placed it near the other steps with same deps, but at the same time it makes sense to place it close to `test.UnitTest` (without race)